test/functional/testlib.bash

```diff
@@ -411,6 +477,7 @@ create_manifest() {
 	local path=$1
 	local name=$2
 	local format=${3:-1}
+	local minversion="0"
```

Comment:
Is this meant to default to '0' but set to $3 if it is provided?

Reply:
I intended to default the manifest minversion to 0 at creation. I thought the format variable was for the format header field.

Reply:
Sorry, I didn't mean to reference format variable, I just meant to ask if you intended this to be set via a user-passed argument.

Reply:
I didn't support variable minversions at creation because it would add complexity to support functionality that I don't think I need for delta manifests tests.

I haven't fully scoped the requirements, but I would need to make sure that all added bundles adhere to the minversion. Currently, my implementation performs the minversion update all at once.

If you want testlib minversion support to be more robust, I could look closer.

Reply:
Nope this is fine for now.